src/renderer/features/networks/NetworkFilter/model/network-filter.ts

```diff
@@ -0,0 +1,38 @@
+import { combine, createEvent, createStore, sample } from 'effector';
+
+import { includes } from '@shared/lib/utils';
+import { networkModel } from '@entities/network';
+
+const componentMounted = createEvent();
+
+const $filterQuery = createStore<string>('');
+const queryChanged = createEvent<string>();
+const queryReset = createEvent();
+
+$filterQuery.on(queryChanged, (_, query) => query).reset(queryReset);
+
+sample({
+  clock: componentMounted,
+  target: queryReset,
+});
+
+const $networksFiltered = combine(
```

Comment:
I'd rather stick to this one
```suggestion
const $filteredNetworks = combine(
```